src/main/resources/assets/js/components/AppListComponent.jsx

```diff
@@ -60,7 +56,10 @@ var AppListComponent = React.createClass({
         </tr>;
     } else {
       appNodes = this.props.collection.map(function (model) {
-        return <AppComponent key={model.id} model={model} onClick={this.onClickApp} />;
+        return <AppComponent
```

Comment:
Could you move this into a separate function, and give the to other states show/hide classSets?